adapters/Pangle/PangleAdapter/GADMediationAdapterPangle.m

```diff
@@ -142,51 +145,49 @@ - (void)loadRewardedAdForAdConfiguration:(GADMediationRewardedAdConfiguration *)
 }
 
 - (void)loadNativeAdForAdConfiguration:(nonnull GADMediationNativeAdConfiguration *)adConfiguration
-                     completionHandler:
-                         (nonnull GADMediationNativeLoadCompletionHandler)completionHandler {
-  [GADMediationAdapterPangle setCOPPA:(adConfiguration.childDirectedTreatment
-                                           ? adConfiguration.childDirectedTreatment.integerValue
-                                           : -1)];
-  _nativeRenderer = [[GADPangleRTBNativeRenderer alloc] init];
-  [_nativeRenderer renderNativeAdForAdConfiguration:adConfiguration
-                                  completionHandler:completionHandler];
+                     completionHandler:(nonnull GADMediationNativeLoadCompletionHandler)completionHandler {
+    [GADMediationAdapterPangle setChildDirected:(adConfiguration.childDirectedTreatment
+                                             ? adConfiguration.childDirectedTreatment.integerValue
+                                             : -1)];
+    _nativeRenderer = [[GADPangleRTBNativeRenderer alloc] init];
+    [_nativeRenderer renderNativeAdForAdConfiguration:adConfiguration completionHandler:completionHandler];
 }
 
 /// Set the COPPA setting in Pangle SDK.
 ///
-/// @param COPPA An integer value that indicates whether the app should be treated as
-/// child-directed for purposes of the COPPA.  0 means false. 1 means true. -1 means
-/// unspecified. Any value outside of -1, 0, or 1 will result in this method being a no-op.
-+ (void)setCOPPA:(NSInteger)COPPA {
-  if (COPPA != 0 && COPPA != 1 && COPPA != -1) {
+/// @param childDirected An integer value that indicates whether the app should be treated as
+/// child-directed for purposes of the COPPA.  See <a
+/// href="https://www.pangleglobal.com/integration/ios-initialize-pangle-sdk">
+/// Pangle's documentation</a> for more information about what values may be provided.
++ (void)setChildDirected:(NSInteger)childDirected {
+  if (childDirected != 0 && childDirected != 1 && childDirected != -1) {
     GADMPangleLog(@"Invalid COPPA value. Pangle SDK only accepts -1, 0 or 1.");
     return;
   }
-  if (BUAdSDKManager.initializationState == BUAdSDKInitializationStateReady) {
-    [BUAdSDKManager setCoppa:COPPA];
+  if (PAGSdk.initializationState == PAGSDKInitializationStateReady) {
+      PAGConfig.shareConfig.childDirected = childDirected;
```

Comment:
We should evaluate the input `COPPA` value before passing to Pangle SDK.

Note that:
- `YES` (or `1`) means the content will be treated as child-directed for purposes of COPPA.
    This should be equivalent to `PAGChildDirectedTypeChild`, but is that `1`?
- `NO` (or `0`) means the content won't be treated as child-directed for purposes of COPPA.
    This should be equivalent to `PAGChildDirectedTypeNonChild `, but is that `0`?

Reply:
Yes.